notebooks/4_SparseKernelMethods.ipynb

```diff
@@ -209,7 +193,7 @@
     "\n",
```

Comment:
This previous statement "It is best..." is no longer relevant given the code below.

Reply:
I think it is still relevant given the code in the cell below the markdown paragraph: the column mean is saved and used to center the test set kernel, just like is advised in the statement.

Reply:
Yes, but it is not being explicitly saved in a variable, as before.

Reply:
It is, the variable is `K_center` (renamed from `barKM`, which I think is what it was before).